base.docker

```diff
@@ -11,6 +12,7 @@ RUN apt-get update && \
     curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" && \
     tar -xJf "node-v$NODE_VERSION-linux-x64.tar.xz" -C /usr/local --strip-components=1 && \
     rm "node-v$NODE_VERSION-linux-x64.tar.xz" && \
+    curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version $YARN_VERSION && \
```

Comment:
Rather than downloading this bash script, can we install yarn via the package manager? https://yarnpkg.com/lang/en/docs/install/#linux-tab

Reply:
I'm not sure we can specify a version using the package manager

Reply:
I've had a bit of a fiddle, and the following works. Can be added to the docker file in the appropriate places.

```
ENV YARN_VERSION 1.3.2-1

apt-get install apt-transport-https && \
curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
apt-get update && \
apt-get install -y --no-install-recommends yarn=${YARN_VERSION} && \
```

Reply:
Nice! That seems like a better way of doing it, I'm changing it now.